src/stk/calculators/base_calculators.py

```diff
@@ -0,0 +1,109 @@
+"""
+Base Calculators
+================
+
+
+
+"""
+
+
+class Calculator:
+    """
+    A base class for all calculators.
+
+    """
+
+    def __init__(self, **kwargs):
+        return
+
+
+class MoleculeCalculator(Calculator):
+    """
+    Base class for calculators which operate on single molecules.
+
+    """
+
+    def __init__(self, use_cache, **kwargs):
+        self._use_cache = use_cache
```

Comment:
Why not have `_use_cache` in calculators if it's just a bool. Can we not assume every calculator is going to use the cache?

Reply:
Indeed not, for exmple PopulationCalculator has no need for cache - for example a FitnessNormalizer